### ISSUE TITLE
Nedbank Bank

### ISSUE THREAD

**Issue by sindile** [alltheplaces/alltheplaces#826]
https://www.nedbank.co.za/content/nedbank/desktop/gt/en/personal/tools-and-guidance/branch-locator.html

**Comment by CloCkWeRX** [comment]
https://personal.nedbank.co.za/contact/find-us.html

**Comment by CloCkWeRX** [comment]
```
curl 'https://api.nedsecure.co.za/nedbank/channeldistribution/v2/branches?carduplift=%22Yes%22&resultsize=10&latitude=-26.10757&longitude=28.05675' \
  -H 'Accept: */*' \
  -H 'Accept-Language: en-GB,en;q=0.9,en-US;q=0.8,de;q=0.7,es;q=0.6,fr;q=0.5' \
  -H 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiIxOGYyMjA4OS05OWY2LTQ4MmUtOTMzMC02N2IwM2RlOTNjZjEiLCJ0b2tlbl90eXBlIjoiQmVhcmVyIiwibmJmIjoxNzAwODE5Nzc2LCJpc3MiOiJpZHAubmVkYmFuay5jby56YSIsImlhdCI6MTcwMDgxOTgzNiwiZXhwIjoxNzMyMzU1ODM2LCJncmFudF90eXBlIjoiYW5vbnltb3VzIiwianRpIjoiZDY3ZDMwZmE2NjFhNGQ1YWExMmIwNTJhMjg0ZTA2M2YiLCJzY29wZXMiOltdfQ.ttJhX9fzFFH6PIpGPhWIuYUl0sFgFQ-rvfiZG8g_SjJFwKwGntbCP9tRBLXiqqhoZzP6MLxHt7kxOg7S_5oPDf6Ug0wqc-PvlkFC5KIe6eXP4j83_S7WB0TkD-sSCOD54iHBr8RR765M04jwZ8PBvv7eIY8LuzgOV6YWFyAgPerIAb9khuDxlkPfTXoczSzns5GO063Un6VcBRvoJdIZwkrJly14rsYknEkNS1ZzJdt7j8POJnZCiXs5Y134R4_6e68_0zLsP4V5VrBp-gzK_PKGBaDOS1DDxNWU9_B_x4IvO5xiCqW88JBaiCwFN6yoWVN5GCKjP-9H1TbvaqGHIg' \
  -H 'Connection: keep-alive' \
  -H 'DNT: 1' \
  -H 'Origin: https://personal.nedbank.co.za' \
  -H 'Referer: https://personal.nedbank.co.za/' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: cross-site' \
  -H 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36' \
  -H 'sec-ch-ua: "Not A(Brand";v="99", "Google Chrome";v="121", "Chromium";v="121"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Windows"' \
  -H 'sec-gpc: 1' \
  --compressed
```

Results otherwise look good

**Comment by arrival-spring** [comment]
Fixed by #10325
